main.js

```diff
@@ -0,0 +1,13 @@
+const tagSpan = document.querySelector('span');

```

Comment:
Do not use tagName in variable names

What if you want to change span to div
Use more abstract variables names